modules/edge-identity/src/identity.rs

```diff
@@ -156,17 +158,53 @@ decl_module! {
 				IdentityStage::Attested => ()
 			}
 
+			// Check the number of verifications the record has and ensure all are valid
+			let mut valid_verifications: Vec<(T::AccountId, bool)> = [(_sender.clone(), vote)].to_vec();
+			if let Some(verifications) = record.verifications {
+				valid_verifications = verifications.clone()
+					.into_iter()
+					.filter(|v| Self::verifiers().contains(&v.0))
+					.collect();
+				valid_verifications.push((_sender.clone(), vote));
+			}
+
+			let yes_votes: Vec<(T::AccountId, bool)> = valid_verifications.clone()
+				.into_iter()
+				.filter(|v| v.1 == true)
+				.collect();
+
+			let no_votes: Vec<(T::AccountId, bool)> = valid_verifications.clone()
```

Comment:
What do we do with no votes?

Reply:
We delete the identity, see the comments above the supermajority check of the no_votes. Since attestations could be false, we also want verifiers to say no on those and delete them immediately. I suppose I may have forgotten to free up the space elsewhere.